collector/filesystem_common.go

```diff
@@ -126,7 +127,15 @@ func (c *filesystemCollector) Update(ch chan<- prometheus.Metric) (err error) {
 	if err != nil {
 		return err
 	}
+	// Make sure we expose a metric once, even if there are multiple mounts
+	updated := map[string]bool{}
 	for _, s := range stats {
+		k := fmt.Sprintf("%q", s)
```

Comment:
Not sure if that's the way to go. Could also just iterate over stats to check if something was printed already. Thoughts?